base/bool.jl

```diff
@@ -77,6 +77,7 @@ sign(x::Bool) = x
 abs(x::Bool) = x
 abs2(x::Bool) = x
 iszero(x::Bool) = !x
+isone(x::Bool) = x
```

Comment:
It's say keep it for completeness and since it makes it clearer to see what's happening.